### DISCUSSION TITLE
Collapsible Sidebar Component

### DISCUSSION THREAD

**Discussion by sayanChaterjee** [saadeghi/daisyui#3240] [Ideas / feature requests]
Hey @saadeghi Can i contribute a sidebar component which will be expandable on hovering. Looking forward to your suggestions.

**Comment by saadeghi** [comment]
Related discussion: https://github.com/saadeghi/daisyui/discussions/2657

>There are many ways to implement this type of collapsible sidebar so we can't simply do it in CSS using a class name.
For example, what should be the sidebar width values, what parts of the menu items should be hidden? if there's a search input on the sidebar how should it look like when collapsed? If there are buttons, logos, links without icon, select box, how should they look like when collapsed?  
It's a very opinionated case 😅
